xmind/tests/test_xmind__init__.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from xmind.tests import base
+from xmind import *
+from unittest.mock import Mock, call
+
+
+class TestXmindInit(base.Base):
+    """Tests for xmind.__init__.py file"""
+
+    def getLogger(self):
+        if not getattr(self, '_logger', None):
+            self._logger = logging.getLogger('xmindInit')
+        return self._logger
+
+    def test_excessive_parameters(self):
+        _workbook_loader = self._init_patch_with_name('_workbook_loader', 'xmind.WorkbookLoader')
+        _workbook_saver = self._init_patch_with_name('_workbook_saver', 'xmind.WorkbookSaver')
+
+        with self.assertRaises(TypeError) as _ex:
```

Comment:
What about check for none parameters during a call of 'load' and 'save'

Reply:
Done